content/docs/contributing/to-docs/templates/component-reference-doc.md

```diff
@@ -0,0 +1,428 @@
+---
+linkTitle: Component Reference Document
+title: Writing an O3DE Component Reference Document
+description: Guidelines for writing a component reference document for Open 3D Engine (O3DE) using the component reference document template. 
+weight: 200
+toc: true
+---
+
+This topic guides you through the writing process for an **Open 3D Engine (O3DE)** component reference document.
+
+The workflow: 
+
+1. [Determine if your document belongs in the O3DE Component Reference](#does-my-document-belong-in-the-o3de-component-reference). 
+2. [Download the component reference document template.](#download-the-component-reference-document-template)
+3. [Follow the writing guidelines to help you fill in the template.](#writing-a-component-reference-document)
+4. [Determine where your component reference document belongs in the o3de.org folder.](#where-does-the-component-reference-document-live)
+5. [Store the images that are in your document in the correct folder.](#storing-image-files)
+6. [Update the O3DE Component Reference to list your document.](#updating-the-o3de-component-reference)
+7. [Submit your document to O3DE Documentation.](#submit-your-component-reference-document)
+
+All documentation must adhere to the standards outlined in the O3DE [Style Guide](/docs/contributing/to-docs/style-guide/)and [Terminology](/docs/contributing/to-docs/terminology/). If you need any help, reach out to the Documentation and Community Special Interest Group (D&C SIG) on [Discord](https://discord.com/invite/o3de). 
+
+
+## Does my document belong in the O3DE Component Reference?
+
+First, determine whether or not your document belongs in the [Component Reference](/docs/user-guide/components/reference/) section of the O3DE User Guide. The purpose of the component reference document is exactly that, a page that users can quickly reference when using the component in **O3DE Editor**.
+
+Supplemental information, such as using the component across different workflows, belongs in other documentation. That document can be organized in an appropriate section of the User Guide, such as the [Interactivity and Simulation](/docs/user-guide/interactivity/), [Scripting](/docs/user-guide/scripting/), [Networking](/docs/user-guide/networking/), or [Visualization](/docs/user-guide/visualization/) sections.
+
+## Download template
```

Comment:
```suggestion
## Set up repository and download component template
```

Suggest that you add some info linking back to the instructions on setting up for contributing to o3de.org (https://www.o3de.org/docs/contributing/to-docs/o3de-docs-repo-setup/)